omnisafe/algorithms/algo_wrapper.py

```diff
@@ -71,7 +77,8 @@ def _init_config(self):
             if 'algo' in self.custom_cfgs:
                 self.custom_cfgs.pop('algo')
             # validate the keys of custom configuration
-            recursive_check_config(self.custom_cfgs, cfgs)
+            if self.algo_type != 'offline':
```

Comment:
Why don ‘t we check config for offline？

Reply:
After the model-based code is merged, I will refactor algo_wrapper.py, I just use special case for now